ext/standard/tests/array/is_countable.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+Check if variables can be counted
+--FILE--
+<?php
+
+$types = [
+    "array"     =>  [],
+    "null"      =>  null,
+    "string"    =>  "test",
+    "integer"   =>  123,
+    "float"     =>  3.14,
+    "true"      =>  true,
+    "false"     =>  false,
+    "object"    =>  (object) [],
+    "simplexml" =>  new SimpleXMLElement("<xml><tag1></tag1><tag2></tag2></xml>"),
```

Comment:
Some of the SPL classes (e.g. ArrayObject and SplFixedArray) also implement count_elements, and are always available as of PHP 5.3.0.